modules/update-cluster-version-object.adoc

```diff
@@ -37,32 +37,46 @@ $ oc adm upgrade --include-not-recommended
 +
 [NOTE]
 ====
-The additional `--include-not-recommended` parameter includes updates that are available but not recommended due to a known risk that applies to the cluster.
+The additional `--include-not-recommended` parameter includes updates that are available with known issues that apply to the cluster.
 ====
 +
 .Example output
 [source,terminal]
 ----
-Cluster version is 4.10.22
+$ oc adm upgrade --include-not-recommended
```

Comment:
This line is already shown in the separate code block above, so it should be removed from this example output block